radixdlt-core/radixdlt/src/main/java/com/radixdlt/RadixNodeModule.java

```diff
@@ -123,11 +125,10 @@
 
   public RadixNodeModule(RuntimeProperties properties) {
     this.properties = properties;
-    var networkId = properties.get("network.id");
-    if (networkId == null) {
-      throw new IllegalStateException("Must specify network.id");
-    }
-    this.networkId = Integer.parseInt(networkId);
+    this.networkId =
+        Optional.ofNullable(properties.get("network.id"))
+            .map(Integer::parseInt)
```

Comment:
This is gonna throw a `NumberFormatException` if the id isn't a parsable integer.

Reply:
Yes. Exactly like original code does.